### PR TITLE
Added region tags for inclusion in C.G.C. Added README

### PR DESCRIPTION
**Pull request by betsy-lichtenberg**
Intending to add to new page similar to https://cloud.google.com/load-balancing/docs/https/ext-http-lb-tf-module-examples, but for L4 ILB.

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

File: examples/minimal/main.tf [hunk 1]
```diff
@@ -64,8 +63,10 @@ resource "google_compute_subnetwork" "test" {
   ip_cidr_range = "10.2.0.0/16"
 }
 
+# [START cloudloadbalancing_int_tcp_udp_gce]
 module "test_ilb" {
-  source       = "../../"
+  source       = "GoogleCloudPlatform/lb-internal/google"
+  version      = "2.3.0"
```
Comment (morgante) [inline examples/minimal/main.tf:69]:
```suggestion
  version      = "~> 4.0"
```
Remember to always check the CHANGELOG for the latest up to date release: https://github.com/terraform-google-modules/terraform-google-lb-internal/blob/master/CHANGELOG.md

**Review by morgante — CHANGES_REQUESTED** [review]

File: examples/minimal/main.tf [hunk 2]
```diff
@@ -15,7 +15,7 @@
  */
 
 provider "google" {
-  version = "~> 3.62.0"
+  version = "~> 3.14.0"
```
Comment (morgante) [inline examples/minimal/main.tf:18]:
```suggestion
  version = "~> 3.62.0"
```

**Review by morgante — CHANGES_REQUESTED** [review]

File: examples/minimal/main.tf [hunk 3]
```diff
@@ -46,7 +46,6 @@ locals {
     request             = ""
     request_path        = "/"
     host                = "1.2.3.4"
-    enable_log          = false
```
Comment (morgante) [inline examples/minimal/main.tf:49]: Why did you remove this line?

**Review by morgante — CHANGES_REQUESTED** [review]
